pages/about.html

```diff
@@ -5,8 +5,71 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="./styles/about.css" rel="stylesheet">
 </head>
+
 <body>
+  <header>
+    <h1>About me!</h1>
+    <p id="my-intro">
+      My name is <b>Pavi Rannulu.</b> 
+      <br>
+      Pronouns:   They/Them/Theirs
+      <br>
+      I am queer, non-binary, brown, and an immigrant.
+      <br>
+      Best known for: having big heart & a curious mind. 
+    </p>
```

Comment:
Consider letting the default styling behavior or `p` tags to break up your lines of text without hardcoding a `br` tag. Each line can be its own `p` tag.